templates/cds_rdm/header.html

```diff
@@ -0,0 +1,18 @@
+{#
+  Copyright (C) 2022 CERN.
+
+  CDS-RDM is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+
+{%- extends "invenio_app_rdm/header.html" %}
+
+{%- block site_banner %}
+  <div class="ui warning message top attached mt-0 " id="temp-banner">
```

Comment:
The problem is CDS logo (should be left-aligned - that is why it looks strange), I would prefer to keep the text of the banner aligned to the left, since we are reading from the left side, it visually gives it more importance

Reply:
I created an issue for this https://github.com/CERNDocumentServer/cds-rdm/issues/43